services/ingestion/etl/cordis/csv/test/stubs/record.json

```diff
@@ -0,0 +1,25 @@
+{
+  "rcn": "215954",
+  "id": "772787",
+  "acronym": "SMARTER",
+  "status": "SIGNED",
```

Comment:
So far, this field has been relatively consistent. Has been available in valor (all 'ongoing') and budg ('ongoing' and 'finalized'). We need to at least lowercase this one.

Reply:
For the dataset I downloaded, we can have `"SIGNED"`, `"CLOSED"`, `"TERMINATED"`

Reply:
Good to ask Laurent whether we could maybe have a fixed set and map through differences. Closed and finalized feel same.